flair/models/word_tagger_model.py

```diff
@@ -108,18 +109,22 @@ def _get_embedding_for_data_point(self, prediction_data_point: Token) -> torch.T
 
     def _get_data_points_from_sentence(self, sentence: Sentence) -> List[Token]:
         # special handling during training if this is a span prediction problem
-        if self.training and self.span_prediction_problem:
+        if self.span_prediction_problem:  # do we need self.training here?
```

Comment:
The conversion is only necessary during training: we take Span labels and encode them as Token-level labels. During prediction, this is not necessary.